src/bentoml/models.py

```diff
@@ -131,20 +136,36 @@ def import_model(
             "diffusers": diffusers_import_model,
         }
 
-        _, repo_name = path.split(":")
-        name = name if name is not None else repo_name.replace("/", "--")
+        model_id = parsedURL.path.strip("/")
+
+        if model_id == "":
+            raise BentoMLException(
+                "Invalid HuggingFace model URL, please check your URL"
+            )
+
+        bento_name = inflection.dasherize(model_id.replace("/", "--"))
```

Comment:
not sure if we depends on inflection. check `pyproject.dependencies` for this.

Reply:
inflection is included in the dependencies list. so should be good 👍